### PR TITLE
New TrackScape Connector plugin

### PR DESCRIPTION
**Pull request by fatfingers23**
Sorry about that going try that again.

This is a new plugin that connects RuneLite to a discord bot called [TrackScape](https://github.com/fatfingers23/trackscape-discord-bot). This bot is used to bridge discords and clan chats. Some features include. 
* Messages in game send to the discord
* Ability to receive in game messages from discord
* Styled broadcasts of various clan broadcasts

### CODE REVIEW

**Comment by raiyni** [comment]
Please don't close and open new prs. Push to the same branch.

**Comment by fatfingers23** [comment]
@raiyni This is the same branch as i pushed to before in #4915 . I closed the previous PR because I had written the commit hash to the wrong plugin, another one that has nothing to do with this plugin. I was using it as a refrence and made a mistaken. I saw it brought in that plugin on the runelite-github-app comment and thought best to create a new one. 

I hard reset my branch to remove the error, and added the proper hash in the correct plugin and made a new PR.

I apologize for the mistake. Please let me know what you would like done to resolve this. Thank you.

**Comment by raiyni** [comment]
You just update the commit hash in the pr...

**Comment by fatfingers23** [comment]
Checked out at the previous hash instead of reseting and removing it and added the fix in a new commit. I assume this is what you mean by using the same branch. Please let me know if there is anything else.

**Comment by Felanbird** [comment]
The `same branch` comment was just his automatic response, you did use the same branch.

As mentioned, to fix your previous PR all you needed to do was change the commit for `name-change-detector` back to the commit currently used on the hub.

**Comment by fatfingers23** [comment]
I apologize and will keep it all to one PR from now on. I had thought a new PR would be less confusing and to just remove my previous mistake. Sorry about all the runaround.

This should have everything set and no wires crossed. I appreciate y'alls time.

File: plugins/trackscape-connector [hunk 1]
```diff
@@ -0,0 +1,3 @@
+repository=https://github.com/fatfingers23/trackscape-connector-plugin.git
+commit=6594ed40172a34c83a3bcd0d51f5434e44baf722
+warning=This plugin may submit all your clan chat messages and your IP address to a 3rd party website not controlled or verified by the RuneLite Developers. As well as may receive incoming messages from said website.
```
Comment (Nightfirecat) [inline plugins/trackscape-connector:3]:
```suggestion
warning=This plugin submits clan chat messages and your IP address to a 3rd party website not controlled or verified by the RuneLite Developers.
```

**Pull request merged**